lib/src/models/curl.dart

```diff
@@ -128,40 +150,51 @@ class Curl extends Equatable {
       }
     }
 
-    String? url = clean(result['url']);
-    final String? data = result['data'];
-    final String? cookie = result['cookie'];
-    final String? user = result['user'];
-    final String? referer = result['referer'];
-    final String? userAgent = result['user-agent'];
-    final bool form = result['form'] ?? false;
-    final bool head = result['head'] ?? false;
-    final bool insecure = result['insecure'] ?? false;
-    final bool location = result['location'] ?? false;
```

Comment:
Revert and use this approach of accessing values and storing in local variables.